themes/vocabulary_theme/templates/macros/authors.html

```diff
@@ -9,7 +9,7 @@
         </figure>
       {% else %}
       <figure class="image profile">
-        <img class="profile" src="./blank_profile.svg" alt="gravatar" />
+        <img class="profile" src="../../../blank_profile.svg" alt="gravatar" />
```

Comment:
As an added benefit, you can remove the `blank_profile.svg` asset.

Reply:
Oh okay! Lemme try.

Reply:
Are you talking about this? 
![image](https://user-images.githubusercontent.com/34674462/90644320-386bb580-e252-11ea-99ed-9e2faf94df7e.png)

Reply:
<img width="929" alt="Screenshot 2020-08-21 at 1 59 40 PM" src="https://user-images.githubusercontent.com/16580576/90869943-90bbc800-e3b6-11ea-8cf6-ac9428690c4c.png">

You can set `d=???` to any of these. Take a call with @panchovm as to which works best.

Reply:
If we go this direction I personally think 'retro' is the cleanest (is also used by lots of other tech sites/platforms)

Reply:
We can get rid of the svg default image I made, and as @TimidRobot said, MP is the better one indicating there is a person behind the post. I will go with MP (mystery person).